### DISCUSSION TITLE
Error Message "Windows protected your PC"

### DISCUSSION THREAD

**Discussion by Daniel-Verbundprojekt** [mifi/lossless-cut#678] [Q&A]
Hi,
I don't know much about this. Just downloaded the latest Version 3.33.1 and when trying to execute it, I get this message. Is that normal, a bug, something wrong with my PC or how should I interpret this?

[https://www.bildhost.com/images/2021/02/22/Unbenannt.png](url)

**Comment by TarkusLV** [comment]
I always get that alert whenever I update to a new version.  I forget the dialog, but I think you click on More Info or something like that, and then Run Anyway.  I think it's something to do with the lack of a certificate.  Not a big deal in this case.

**Comment by mifi** [comment]
Yea like @TarkusLV said, the losslesscut binary from github is not signed, so windows will complain. You can either ignore the warning and continue using the free version or you can  buy a signed version in the Windows Store https://www.microsoft.com/en-us/p/losslesscut/9p30lsr4705l (and support my work)
